src/main/scala/LongShort.scala

```diff
@@ -0,0 +1,32 @@
+
+/**
```

Comment:
Personally I delete these creation comments, but if you don't mind them leave them be :)